src/firmware_new/tools/send_proto_over_udp.cpp

```diff
@@ -36,21 +36,20 @@ int main(int argc, char* argv[])
 
     // create a RobotCommunicator with a NetworkMedium
     RobotCommunicator<control_msg, robot_ack> communicator(
-        std::make_unique<NetworkMedium>("192.168.1.140", 42069),
+        std::make_unique<NetworkMedium>("ff02::c3d0:42d2:bb8%eth0", 42000, 42001),
         [&](const control_msg& msg) {
             std::cout << "COMP Txed " << count++ << " msgs " << std::endl;
         },
         [&](const robot_ack& msg) {
             std::cout << "STM32 Rxed " << msg.msg_count() << " msgs " << std::endl;
         });
 
-
     while (1)
     {
         communicator.send_proto(control_req);
 
         // 4000 hz test
-        std::this_thread::sleep_for(std::chrono::nanoseconds(250000));
+        std::this_thread::sleep_for(std::chrono::seconds(1));
```

Comment:
Should the sleep period be changed back back to 250000 ns?

Reply:
reverted, this is more of just a test file